packages/fxa-auth-server/lib/senders/emails/layouts/fxa/en.ftl

```diff
@@ -1,3 +1,8 @@
+
+## Email content
```

Comment:
Just a heads up that this is not guaranteed to be the first file concatenated to `auth.ftl`. If we notice in the future due to a new file that this is no longer grouping correctly we might want to move it back to `auth.ftl`.